module-2/backend/week_two.md

```diff
@@ -5,21 +5,43 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  `ActiveRecord is a object relationship manager which allows you to create relational databases with built in functions you' can call on them.`
+  
 2. What is a migration?
+  `A migration is the structure of the database which allows you to add and edit tables and ultimately allows that entire database to be recreated`
+  
 3. How does a table relate to a model?
+  `A model communicates with the table in a database, the model contains methods which can be called on the database and allows us to draw relationships between tables`
+  
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  `They are class methods, they are called on the class and not on instances of the class`
+  
 5. What do they allow you to do?
+  'They allow you to draw relationships between different tables.  For example populating a house table with id's which point to a table which contains each individual room'
```

Comment:
Just checking: in this case I think the foreign key would be on the `rooms` table. E.g.:

**house**
* address

**room**
* name
* house_id